aviatrix/provider.go

```diff
@@ -42,6 +44,30 @@ func Provider() *schema.Provider {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"ignore_tags": {
```

Comment:
Is it a good name?

Reply:
This is what aws uses on their provider.